Sources/SourceKitLSP/Swift/SwiftLanguageServer.swift

```diff
@@ -1037,6 +1037,34 @@ extension SwiftLanguageServer {
     completion(.success(codeActions))
   }
 
+  public func inlayHints(_ req: Request<InlayHintsRequest>) {
+    let uri = req.params.textDocument.uri
+    expressionTypeInfos(uri) { result in
+      switch result {
+      case .success(let infos):
+        let hints = infos
```

Comment:
Probably worth doing `.lazy.map` to avoid the intermediate array since this could be a large array